Library/Formula/couchpotatoserver.rb

```diff
@@ -54,4 +54,8 @@ def startup_script; <<-EOS.undent
   def caveats
     "CouchPotatoServer defaults to port 5050."
   end
+
+  test do
+    system "#{bin}/couchpotatoserver", "-h"
```

Comment:
It's better than nothing, but it is possible to write a test that checks some more of the functionality of the program?